### PR TITLE
Fix-Input widget with text area expands over boundaries

### LINKED ISSUE #1001: [Bug] Input widget expands without boundaries in published mode
https://www.notion.so/Input-widget-expands-without-boundaries-in-published-mode-33d0ceaae8ba4ec09a0d94d50dfe6963

### PR DESCRIPTION
**Pull request by vicky-primathon**
## Description
Fix Input widget with text area expands over boundaries

Fixes #1001 

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?
- Cypress test that validates resize property is fixed

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:green_circle: Total coverage has increased</summary>

    // Code coverage diff between base branch:release and head branch: fix/input-widget-textarea-resize 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :green_circle: | total | 54.13 **(0)** | 36.14 **(0.02)** | 32.8 **(0.01)** | 54.63 **(0)**
 :green_circle: | app/client/src/components/designSystems/blueprint/InputComponent.tsx | 63.01 **(1.37)** | 41.94 **(4.84)** | 51.72 **(3.44)** | 62.32 **(0)**</details>

### CODE REVIEW

**Review by somangshu — APPROVED** [review]
Approving to run tests, Looks good on Deploy preview

File: app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/FormWidgets/Input_textarea_spec.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const dsl = require("../../../../fixtures/InputTextAreaDsl.json");
```
Comment (hetunandu) [inline app/client/cypress/integration/Smoke_TestSuite/ClientSideTests/FormWidgets/Input_textarea_spec.js:1]: Please don't test for CSS in cypress tests. Use jest tests instead

**Review by hetunandu — CHANGES_REQUESTED** [review]

**Review by somangshu — APPROVED** [review]

**Comment by somangshu** [comment]
Unrelated tests failing here again!

**Review by hetunandu — APPROVED** [review]

**Review by mohanarpit — APPROVED** [review]

**Comment by somangshu** [comment]
Rerunning all jobs - Unrelated tests failing

**Review by somangshu — APPROVED** [review]

**Pull request merged**